### DISCUSSION TITLE
SCRUM PREP MILESTONE 2-4

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#234] [General]
held weekly meetings, scrum prep a checked documentation

**Comment by Saul-Caban-UPRM** [comment]
Finished task #226 for the week. Had meeting with team members

**Comment by Joaquin939** [comment]
This week I was looking for some new issues for the end of this milestone.

**Comment by AngelCIICMorales** [comment]
Held weekly meetings and  reviewed tasks.

**Comment by Negroni1** [comment]
Applied professor's milestone 1 feedback on the synopsis and domain requirements to the documentation.

**Comment by FrancesC8** [comment]
Not much progress this week, but plan to still working on refactoring tasks to adopt SOLID principles in backend #147 & #145. This weekend I will send out the PRs for both tasks.

**Comment by print-Eruki** [comment]
UUID unit test in progress, finalizing fixing guidelines in user stories

**Comment by JadiellMV** [comment]
This week I kept working on the password reset feature.

**Comment by AnthonyMed13** [comment]
Worked on documentation tasks and debug some code.

**Comment by sandy-marrero** [comment]
- ⁠Refactored the login form to integrate the SOLID principles #173 
- ⁠Updated the Login page UI to match the new homepage styling #232 
- ⁠Starting work on fixing the Function Signatures to address Milestone 1 feedback #195

**Comment by ebrain-mendoza121** [comment]
Attended meetings, worked on lobby task and documentation

**Comment by DavidCuevas3** [comment]
Attended the weekly meeting and brainstormed on possible ways to solve the assigned task. #171

**Comment by rukitoyordan** [comment]
Changed the palette of 4 new backgrounds for the game to align with the desired colors

**Comment by ianramospr** [comment]
I was looking for some new issues for the end of the milestone.

**Comment by fernandopizarro1** [comment]
This week I finished the Advanced Bot, now working out some extra kinks. Found some bugs and fixed them along with my team.

**Comment by nsfrontera** [comment]
Finishing the all five game mode and working on other tasks

**Comment by ilikecheese117** [comment]
Working on issues for the end of milestone 2.

**Comment by Adriana-Isabel** [comment]
Working on the Home Enhancements issue #225 .

**Comment by Gaby-Concepcion** [comment]
Worked on game mode tasks and documentation review.

**Comment by PinkSylvie** [comment]
Worked on identifying and defining our project’s aggregates and invariants for the documentation

**Comment by Jvstinoo** [comment]
Working on finishing my tasks such as adding additional unit testing.

**Comment by G4vp** [comment]
Working on finishing my lecture topic tasks and gameplay tasks.

**Comment by RicardoJRSs** [comment]
Working on documentation and tasks

**Comment by frances-sepulveda** [comment]
Worked with @rukitoyordan to continue improving the project aesthetics
